src/windows.rs

```diff
@@ -15,15 +15,42 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
+
+#![allow(unsafe_code)]
```

Comment:
We tend to avoid using it at this scope but rather limit it to only the function using the unsafe block (unless ofc all functions in the module are unsafe, which is not the case here).